### ISSUE TITLE
use dockerslim on containers?

### ISSUE THREAD

**Issue by chrillep** [lando/lando#1968]
I just found this and i'm not sure if its applicable but sure seems nice! =)

https://dockersl.im/
Minify and Secure Your Docker Containers. Frictionless!

**Comment by pirog** [comment]
@chrillep interesting. this definitely seems like something worth investigating more. 

my concern would be that it renders the container not very useful for local dev purposes, eg i think people like being able to ssh into the containers and install packages, hack around. Could certainly be a toggle option for slim vs something else though. 

@dustinleblanc weve got bigger fish to fry right now but this might be something youd be interested in investigating down the road

**Comment by chrillep** [comment]
I agree! 😊 i just found it amazing how small you can get the images.
I you ever plan on taking lando to production this could be useful ✌️. 

i haven't tried it in a pipeline but could be worth a try. setting up lando  as a main docker host and deploying to it with slim images 😱

Thanks for great software!

**Comment by pirog** [comment]
@chrillep i think in a hypothetical lando 4.x we'd allow users to choose between alpine or debian containers.
